news/2025/2025-01-31-locus.md

```diff
@@ -0,0 +1,69 @@
+
+To be specific, all entries are expected to showcase:
+
+- An original song of length 1:30–3:00.
+- A full beatmap spread on the osu! game mode that satisfies the ranking criteria.
+- Visual assets (e.g. background/storyboarding/video).
+
+The full list of rules and operational procedures can be found [here](/wiki/Contests/Locus/2025), but we felt that the following points deserved particular attention and highlighting:
+
+- **All entries must remain anonymous until the contest concludes.** Groups will be able to choose a team name and enter under that pseudonym, and must not share anything related to their creation via social media or other spaces — this is to help ensure each entry is judged by the community solely on its merit, and not by the fame of its creators.
+- **Incomplete entries are still considered valid for submission.** They will undergo the same voting process as every other entry. We want to encourage all creators to participate, even if the full scope of their creation isn't achieved.
+- **The ranking of the entries will be determined by a new form of community voting (elo system).** Voters will be required to compare a set number of entries across 3 criteria and the results will be fed into an elo rating system to determine the final rankings.
+  - The process is similar to a round-robin tournament where every player is matched against each other multiple times, except the match outcomes are decided by the voters, essentially taken as "wins" or "losses" and fed into an elo rating system.
+  - This is, in spirit, similar to ranking systems used by games like chess or League of Legends, and was constructed to make the voting more accessible to the general community (by not having to go through every single entry to participate in voting), and to help prevent bandwagoning.
+
+### Schedule and registration
+
+| Event | Date |
+| --: | :-- |
+| Entry submission deadline | 1 June 2025 |
+| Community voting period | June 2025 |
+| Results stream | Early July 2025 |
+
+Teams must be registered in order to submit an entry, so we recommend doing that once your team is formed in order for us to contact you with updates throughout the competition.
+
+[**Click here to register for this contest!**](https://form-auth.ppy.sh/form/2025-locus-registrations)
+
+### Prizes
+
+For a contest at this level and scope, we have decided to offer equal rewards for the top entries:
+
+- The beatmap will be bundled as part of the default osu! installation as "recommended beatmaps" for at least 1 year.
+- Team members will receive a unique profile badge themed around their entry.
+- Team members will be awarded a *Locus* user title befitting of their role.
+
+For any complete entry submitted before the entry submission deadline, team members will be awarded a profile badge if they are able to rank the set before the next iteration of the contest (they will be prioritised on Mappers' Guild quests).
```

Comment:
```suggestion
For any complete entry submitted before the entry submission deadline, **team members will be awarded a profile badge if they are able to rank the set before the next iteration of the contest**.
```
this is probably the most interesting prize for the contest and it's stuck in the middle of a paragraph :(

i also dont think the quest reference is necessary (it'll happen, but doesnt need mentioning)